src/js/tabs/pid_tuning.js

```diff
@@ -182,8 +182,12 @@ TABS.pid_tuning.initialize = function (callback) {
                 var checked = $(this).is(':checked');
                 if (checked) {
                     const MAX_ACCELERATOR_GAIN = semver.gte(CONFIG.apiVersion, API_VERSION_1_43) ? 3.5 : 1.1;
-                    const itermAcceleratorGain = Math.max(ADVANCED_TUNING.itermAcceleratorGain / 1000, MAX_ACCELERATOR_GAIN);
-                    $('.antigravity input[name="itermAcceleratorGain"]').val(itermAcceleratorGain);
+                    const itermAcceleratorGain = Math.max(ADVANCED_TUNING.itermAcceleratorGain / 1000);
```

Comment:
Maybe you forgot the second parameter to the `Math.max` function? Or you forgot to remove it completely? I suppose the second one...

Reply:
yes sorry, I forgot to remove the math, fixed all  issues now :)